STANDARDS.md

```diff
@@ -1,16 +1,66 @@
-# Dataset Standards
 
-This file outlines the standards and requirements needed for submitting a dataset to QCArchive.
+This file outlines the standards and data requirements needed for submitting a dataset to QCArchive.
 This ensures that we have a consistent data model for downstream processes.
 
-## Required fields 
+STANDARDS version: 3-DRAFT (2020-11-13)
+
+We distinguish between standards for the datasets (i.e. the actual data), and the standards for training force fields.
 
-Current list:
-* Ensure all submissions have cmiles, most important are mapped hydrogen smiles
-* Ensure the WBO is requested for all submissions, this should be included in the scf properties list using the flag `wiberg_lowdin_indices`
+# Dataset Standards
 
-## Best practices
-* If any calculations are to be redone from another collection, re-use the old input (coordinates, atom ordering etc) as this will avoid running the calculation again and will just create new references in the database to the old results and should help keep the cost of the calculations down.  
+## Each molecule must have the following information:
+- Canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of SMILES generation (NEW)
+- Coordinates
+- Total Charge
+- Coordinates must be in CMILES order (discuss)
+
+## Each dataset must have the following information:
+- Name
+- Version (NEW)
+- A short description
+- A long description
+- A link/URL/reference pointing to the provenence to reproduce dataset (e.g. the GH submission folder)
+- A changelog (NEW)
+- A github submitter username
+- The name of the person who selected/sourced the molecules (NEW)
+- A description of the meaning of the entry/molecule keys/names (NEW)
+- Each entry has canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of CMILES generation (NEW)
+- A set of elements that the dataset contains
+- A set of charges that the dataset contains (NEW)
+- The mean and max molecular weight of molecules the dataset contains (NEW)
+- Enumerated stereo flag (True/False) (NEW)
```

Comment:
I think these kinds of tags are really useful to get an idea at a glance of what went into making the dataset, and I think that this list of options has the potential to grow so maybe it would be better to have any, and all workflow stages used in QCSubmit printed here? The settings could also be printed but again this will be in the git repo here.

Reply:
We prevent growing by explicitly stating what is in the standards :) Which ones do we want in this version? I am thinking stereo is stable and ready to use. I don't know whether tautomer is ready for official inclusion.

With stereo enumeration, I am trying to communicate patterns in the dataset. If I have a SMILES, does that mean it's enantiomer is also in the dataset somewhere? This flag indicates that. I don't know if indicating these were fragmented, for example, would convey any useful information in this regard (unless, of course, the dataset contained both the parent and the fragments, in which case a fragment flag would be useful).

Also, just putting all workflow components in this standard would mean forming a standards on filtering and that structure. I am thinking this could possibly come in a standards version later down the road.

Reply:
@j-wags could you possibly comment on perspectives and issues that may warrant not including stereo and tautomer enumeration flags in the datasets?